src/components/expanded-state/ChartExpandedState.js

```diff
@@ -168,6 +168,9 @@ export default function ChartExpandedState({ asset }) {
   if (duration.current === 0) {
     duration.current = 300;
   }
+  const ChartExpandedStateSheetHeight =
```

Comment:
How it possible that this var matters? Don't you have a name collision?

Also, will it work with the case when charts are initially not visible, then fetched after e.g. 5 seconds? The reason I made it not dynamically adjusting the size is that we're not able to modify it after opening (at least for now)